arch/arm/src/nrf52/nrf52_serial.c

```diff
@@ -432,7 +454,118 @@ static int nrf52_interrupt(int irq, void *context, FAR void *arg)
 
 static int nrf52_ioctl(struct file *filep, int cmd, unsigned long arg)
 {
-  return -ENOTTY;
+  struct inode         *inode  = filep->f_inode;
+  struct uart_dev_s    *dev    = inode->i_private;
+  struct nrf52_dev_s   *priv   = (struct nrf52_dev_s *)dev->priv;
+  struct uart_config_s *config = &priv->config;
```

Comment:
The error is on this config variable, maybe add an UNUSED(config) at end of the ioctl() or put the definition inside the #ifdef ...

Reply:
done